tsconfig.json

```diff
@@ -3,7 +3,7 @@
     // we can explicitly declare `any`, but we don't want to infer `any`
     "noImplicitAny": false,
     // maybe bump to 'esNext'?
-    "target": "es6",
+    "target": "es2018",
```

Comment:
how big a breaking change is this?

Reply:
And perhaps more importantly, why it's needed?

Reply:
a possible suggestion from https://github.com/adobe/react-spectrum/issues/6366#issue-2291032648
not sure the least breaking change way to go about this one, but also still have nice syntax

Reply:
Ah, in this case, it sounds like we were already "targetting" ES2018, only unknowingly :D Then I suppose it's not going to be a huge deal, still, worth a place in the release notes for sure.

Reply:
It's worth noting that Parcel doesn't really honour the tsconfig target anyway in its transpilation, it just aims for compatibility with the set of browsers resolved by browserslist.

Since none of the versions on that list is older than 2019, this seems like it should have zero repercussions.